src/components/Description/index.jsx

```diff
@@ -1,7 +1,7 @@
 
 export default function Description(){
     return (<div className={"container m-auto px-[5rem]"}>
-                    <div className={"mx-[5rem] min-w-full"}>
+                    <div className={"text-center sm:mx-[5rem] sm:min-w-full sm:text-left"}>
```

Comment:
Don't know why this is showing up as changed, I reverted my prior commit, this should be unchanged file